extensions/AzureCosmosDBMongoVCore/AzureCosmosDBMongoVCore/AzureCosmosDBMongoVCoreConfig.cs

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+namespace Microsoft.KernelMemory;
+
+public class AzureCosmosDBMongoVCoreConfig
+{
+    /// <summary>
+    /// Get more details about Azure Cosmos Mongo vCore https://learn.microsoft.com/en-us/azure/cosmos-db/mongodb/vcore/vector-search
+    /// </summary>
```

Comment:
this comment seems out of place, maybe a property is missing, or it should be above the class declaration?

Reply:
Fixed.